src/lib/viewers/controls/settings/SettingsDropdown.scss

```diff
@@ -11,7 +11,7 @@ $bp-SettingsDropdown-spacing: 5px;
 }
 
 .bp-SettingsDropdown-flyout {
-    top: initial;
+    top: auto;
```

Comment:
Does this affect the media controls at all?

Reply:
No this is an override only for `SettingsDropdown`